### PR TITLE
docs: fix rook version update command

### PR DESCRIPTION
**Pull request by niels-s**
Fixes some typo's in the v0.4.0 release notes for the Rook Ceph upgrade
steps

### CODE REVIEW

**Review by invidian — APPROVED** [review]
LGTM, thanks @niels-s

File: CHANGELOG.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ https://rook.io/docs/rook/master/ceph-upgrade.html.
   the components:
 
   ```
-  watch --exec kubectl -n rook get deployments -l rook_cluster=rook -o jsonpath='  {range .items[*]}{.metadata.name}{"  \treq/upd/avl: "}{.spec.replicas}{"/"}{.  status.updatedReplicas}{"/"}{.status.readyReplicas}{"  \trook-version="}{.metadata.  labels.rook-version}{"\n"}{end}'
+  watch --exec kubectl -n rook get deployments -l rook_cluster=rook -o jsonpath='{range .items[*]}{.metadata.name}{"  \treq/upd/avl: "}{.spec.replicas}{"/"}{.status.updatedReplicas}{"/"}{.status.readyReplicas}{"  \trook-version="}{.metadata.labels.rook-version}{"\n"}{end}'
```
Comment (johananl) [inline CHANGELOG.md:253]: Any idea if the space before `\t` in `{"  \treq/upd/avl: "}` should be there?

Comment (niels-s) [reply CHANGELOG.md:253]: Yes, I just verified, that the output is nicer with the extra spaces

**Review by surajssd — COMMENTED** [review]
LGTM

**Review by surajssd — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (johananl) [reply CHANGELOG.md:253]: Cool. Thanks for spotting this!

**Pull request merged**